src/app/profile/spaces/space.service.ts

```diff
@@ -108,6 +108,8 @@ export class SpaceService {
   private buildSpaceIndexMap(): void {
     this.spaces.forEach((space, index) => {
       this.spaceIdIndexMap[space.id] = index;
+      // TODO Hack in the path
+      space.path = space.attributes.name;
```

Comment:
I'll remove it.

@VineetReynolds what I would really like is a way to use the typescript get property syntax in model's but I can't find out how to do that!